pinchito/client.go

```diff
@@ -4,11 +4,15 @@ import (
 	"encoding/json"
 	"net/http"
 	"net/url"
+	"errors"
+	"bytes"
+	"os"
 )
 
 func Tapeta() (Log, error) {
+	host := os.Getenv("PINCHITO_HOST")
```

Comment:
I'd do the same here. Maybe make these methods part of the `Client` type and pass in the `PINCHITO_HOST` as a value when initialising it